app/Http/Controllers/AjaxController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use App\Http\Requests;
+
+use App\User;
+use App\profile;
+
+use Log;
+use Carbon\Carbon;
+class AjaxController extends Controller
```

Comment:
Laravel has web and api routes, this lets us name controllers based on exact functionality, AjaxController.php is confusing, it feels like all APIs will be here regardless of the entity. 

Should name this ProfileController.php or something similar, you can specify ajax or api in the controller action/method name to indicate that this is used to process ajax/api requests